edge/client/portainer_edge_async_client.go

```diff
@@ -158,16 +155,6 @@ func (client *PortainerAsyncClient) GetEnvironmentStatus(flags ...string) (*Poll
 			payload.Snapshot.Docker = dockerSnapshot
 			currentSnapshot.Docker = dockerSnapshot
 
-			if client.lastSnapshot.Docker != nil {
-				dockerPatch, err := jsondiff.Compare(client.lastSnapshot.Docker, dockerSnapshot)
-				if err == nil {
-					payload.Snapshot.DockerPatch = dockerPatch
-					payload.Snapshot.Docker = nil
-				} else {
-					log.Printf("[WARN] [edge,client] [message: could not generate the Docker snapshot patch: %s]", err)
-				}
-			}
-
```

Comment:
Hey there's a bunch of removed code in this PR. Was this an accident? It doesn't seem like the other version bump PRs normally change anything other than the version number.

Reply:
actually, base branch was `develop` by mistake. change it to `release/2.15`